### PR TITLE
Updated README with Firefox add-on link

### PR DESCRIPTION
**Pull request by henriquemacedo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 A Chrome extension that brings the simplicity of Google Inbox to Gmail.
 
 ## How to use Simplify
-Add the extension and refresh Gmail: <https://chrome.google.com/webstore/detail/simplify-gmail/pbmlfaiicoikhdbjagjbglnbfcbcojpj>
+Add the extension and refresh Gmail:
+
+- [Chrome](https://chrome.google.com/webstore/detail/simplify-gmail/pbmlfaiicoikhdbjagjbglnbfcbcojpj)
+- [Firefox](https://addons.mozilla.org/pt-PT/firefox/addon/simplifygmail/)
```
Comment (vgladiyenko) [inline README.md]: I guess it's better to use en-US here instead of pt-PT as most people would at least understand EN 🙄

Comment (henriquemacedo) [reply README.md]: You're right! These auto localization URLs are tricky!

Comment (leggett) [reply README.md]: https://addons.mozilla.org/firefox/addon/simplifygmail/ will auto detect your local.

**Comment by leggett** [comment]
Thanks for calling this out. I've added the link for Firefox without the local so it auto-detects.

**Pull request closed without merging**